README.md

```diff
@@ -169,6 +170,20 @@ trait `blue` is set, or both `os=fedora` and `arch=x64` are set.
        ]
 ```
 
+- `timeout`
+
+This is a number, that when set higher than zero, sets the timeout
+for a specific test. The value is specified in seconds.
+
+Example:
+
+A test with the following `timeout` will be allowed to run for
+10 minutes.
+
+```
+      "timeout": 600
+```
```

Comment:
What do you think about making this some sort of multiplier (or addition?) so it combines with the overall timeout?

We have various types of systems, some super fast and some super slow (in terms of cpu and network performance). We use the ooverall timeout value to indicate how fast the overall test suite it, roughly. It seems weird to have to override the individual test values as well if we are overriding the global default.

Reply:
I'm fine with a multiplier. I'll update the PR.

Reply:
I've replaced the `timeout` property with a `timeoutMultiplier` property. @omajid, ptal.

Reply:
Thanks, this looks great!